Modules/Filtering/FastMarching/test/itkFastMarchingUpwindGradientTest.cxx

```diff
@@ -388,7 +388,9 @@ itkFastMarchingUpwindGradientTest(int, char *[])
   };
   for (const auto & ee : allConditions)
   {
-    std::cout << "STREAMED ENUM VALUE FastMarchingUpwindGradientImageFilterEnums::TargetCondition: " << ee << std::endl;
+    std::ostringstream ss;
+    ss << ee;
+    ITK_TEST_EXPECT_TRUE(ss.str().find("INVALID") == std::string::npos);
```

Comment:
If this way to print the enum values is preferred over the other one, it should be put to the SW Guide.

Reply:
I used the output operator to render the enum  into a string so that the results could be validated. Not sure it's a different way to print enums that needs to documented. Just an approach for testing and validation.

Reply:
I'm fine with the approach; I just think that if we now prefer this approach then we should put it into the SW Guide. Thanks Brad.

Reply:
> Does the SWG contain recommendations for how to test enum to to string conversions? 

It does not I'm afraid, but would be good to add it.

> Otherwise this approach is specific to this use case and too specific for the SWG.

I think the approach is valid for all enums. Not sure why it shouldn't.

I do not mean it has to be done now, but I think it is good to have for the sake of consistency. Anyways, it is not a show-stopper for this PR to be merged.